[2024-12-13T21:28:52.922+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-12-13T21:28:52.946+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_data.insert_data.insert_data_program manual__2024-12-13T21:28:42.842267+00:00 [queued]>
[2024-12-13T21:28:52.955+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_data.insert_data.insert_data_program manual__2024-12-13T21:28:42.842267+00:00 [queued]>
[2024-12-13T21:28:52.955+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-12-13T21:28:53.220+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): insert_data.insert_data_program> on 2024-12-13 21:28:42.842267+00:00
[2024-12-13T21:28:53.236+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=7570) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-13T21:28:53.244+0000] {standard_task_runner.py:63} INFO - Started process 7572 to run task
[2024-12-13T21:28:53.255+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_data', 'insert_data.insert_data_program', 'manual__2024-12-13T21:28:42.842267+00:00', '--job-id', '1017', '--raw', '--subdir', 'DAGS_FOLDER/etl_data.py', '--cfg-path', '/tmp/tmpmzf9imfh']
[2024-12-13T21:28:53.258+0000] {standard_task_runner.py:91} INFO - Job 1017: Subtask insert_data.insert_data_program
[2024-12-13T21:28:53.383+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_data.insert_data.insert_data_program manual__2024-12-13T21:28:42.842267+00:00 [running]> on host 1fe973f28a1c
[2024-12-13T21:28:53.469+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_data' AIRFLOW_CTX_TASK_ID='insert_data.insert_data_program' AIRFLOW_CTX_EXECUTION_DATE='2024-12-13T21:28:42.842267+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-13T21:28:42.842267+00:00'
[2024-12-13T21:28:53.470+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-12-13T21:28:53.792+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-12-13T21:28:53.794+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-12-13T21:28:53.815+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_data, task_id=insert_data.insert_data_program, run_id=manual__2024-12-13T21:28:42.842267+00:00, execution_date=20241213T212842, start_date=20241213T212852, end_date=20241213T212853
[2024-12-13T21:28:53.874+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-12-13T21:28:53.917+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-13T21:28:53.918+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
